Description: not define reminder actions in the project file
as there are enough calls for donations on the project website.
Tehre is no need to have anything like that in the application
help menu
Author: Alf Gaida <agaida@siduction.org>
Last-Update: <2015-04-26>
Bug-Debian: http://bugs.debian.org/770359
 
--- a/src/smplayer.pro
+++ b/src/smplayer.pro
@@ -24,7 +24,7 @@ DEFINES += SKINS
 DEFINES += MPRIS2
 DEFINES += UPDATE_CHECKER
 DEFINES += CHECK_UPGRADED
-DEFINES += REMINDER_ACTIONS
+#DEFINES += REMINDER_ACTIONS
 DEFINES += SHAREWIDGET
 #DEFINES += USE_FONTCONFIG_OPTIONS
 DEFINES += AUTO_SHUTDOWN_PC
